### PR TITLE
clearing up the page on private transactions so its easy to follow with the quickstart

### PR DESCRIPTION
**Pull request by joshuafernandes**
## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [ ] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [ ] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [ ] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [ ] you fixed all the issues raised by the tests, if any.
- [ ] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

- Clearing up the example page on private transactions so its easier to follow with the quickstart
- Added some info to the quickstart page so it highlights what's available.

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

### CODE REVIEW

File: docs/Tutorials/Examples/Private-Network-Example.md [hunk 1]
```diff
@@ -271,18 +271,17 @@ Refresh the Block Explorer page in your browser displaying the target test accou
 
 The updated balance reflects the transaction completed using MetaMask.
 
-## Smart contract and dapp usage
+## Smart contract and DApp usage
```
Comment (bgravenorst) [inline docs/Tutorials/Examples/Private-Network-Example.md:274]: I'll raise an issue to make our usage of "dapp" consistent throughout the docs.

File: docs/Tutorials/Examples/Privacy-Example.md [hunk 2]
```diff
@@ -97,87 +83,82 @@ For more information on the endpoints and services, refer to README.md in the in
...
 
 ```bash
-node example/eventEmitter.js
-```
+Member1Besu RPC: http://localhost:20000
+Member1Tessera: http://localhost:9081
 
-!!! tip
+Member2Besu RPC: http://localhost:20002
+Member2Tessera: http://localhost:9082
```
Comment (bgravenorst) [inline docs/Tutorials/Examples/Privacy-Example.md:103]: @joshuafernandes I updated the member names for the Tessera nodes. Let me know if this was the correct thing to do.

Comment (joshuafernandes) [reply docs/Tutorials/Examples/Privacy-Example.md:102]: Yep, good catch :) Nice spotting @bgravenorst !

File: docs/Tutorials/Examples/Privacy-Example.md [hunk 3]
```diff
@@ -97,87 +83,79 @@ For more information on the endpoints and services, refer to README.md in the in
...
 
 ```bash
-node example/eventEmitter.js
-```
+Member1Besu RPC: http://localhost:20000
+Member1Tessera: http://localhost:9081
 
-!!! tip
+Member2Besu RPC: http://localhost:20002
+Member1Tessera: http://localhost:9082
```
[the remark this answers is not in the document]
Comment (joshuafernandes) [reply docs/Tutorials/Examples/Privacy-Example.md]: Thank you for fixing this @bgravenorst :)

**Review by bgravenorst — APPROVED** [review]

**Pull request merged**